beta-src/src/components/ui/WDCountryTable.tsx

```diff
@@ -0,0 +1,177 @@
+    return {
+      [`&.${tableCellClasses.head}`]: {
+        lineHeight: "unset",
+        borderBottom: 0,
+        padding,
+      },
+      [`&.${tableCellClasses.body}`]: {
+        borderBottom: 0,
+        padding,
+      },
+    };
+  });
+  return (
+    <div>
+      <Table stickyHeader size="small" aria-label="country info table">
+        <TableHead sx={{ height: "70px" }}>
+          <TableRow sx={{ verticalAlign: "top" }}>
+            {columns.map((column) => (
+              <WDTableCell key={column.id} align={column.align}>
+                {column.icon && <column.icon />}
+                <Box sx={{ fontSize: 10, fontWeight: 400 }}>
+                  {column.label.toUpperCase()}
+                </Box>
+              </WDTableCell>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {countries.map((country) => (
+            <React.Fragment key={country.power}>
+              <TableRow>
+                {columns.map((column) => {
+                  const style = {
+                    color: "black",
+                    fontWeight: 400,
+                  };
+                  let value: string;
+                  switch (column.id) {
+                    case "power":
+                      value = mobileLandscapeLayout
+                        ? country.abbr.toUpperCase()
+                        : country.power.toUpperCase();
+                      style.color = country.color;
+                      style.fontWeight = 700;
+                      break;
+                    case "delaysLeft":
+                      value = `${country[column.id]}/${maxDelays}`;
+                      break;
+                    default:
+                      value = `${country[column.id]}`;
+                      break;
+                  }
+                  return (
+                    <WDTableCell key={column.id} align={column.align}>
+                      <span style={style}>{value}</span>
+                    </WDTableCell>
+                  );
+                })}
+              </TableRow>
+              {country.votes && (
+                <TableRow>
+                  <WDTableCell
+                    sx={{
+                      fontSize: "70%",
+                      paddingTop: "0px !important",
+                      fontWeight: 700,
+                    }}
+                    colSpan={columns.length}
+                  >
+                    <Box
+                      sx={{
+                        color: "#959595",
+                        display: "inline-block",
+                        marginRight: "10px",
+                      }}
+                    >
+                      VOTED
+                    </Box>
+                    {Object.entries(country.votes)
+                      .filter((data) => {
+                        return data[1];
+                      })
+                      .map((data) => {
+                        return (
```

Comment:
instead of filtering out PAUSE, let's just return a chip when the vote isn't PAUSE

Reply:
not following, `data[1]` is a boolean

Reply:
ok, maybe i'm not following this code. you're getting the entries in the votes object and filtering the values that are true into an array that you are mapping over? we can reduce the looping by just making the map return null with conditional rendering.

vote && <Chip....